Dockerfile

```diff
@@ -1,15 +1,21 @@
 FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
 
-ENV SOLR_VERSION 7.7.2
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
```

Comment:
What is this version number? I thought the label would be for the version of the app that is being run by the container (so in this case it would be "7.7.2")...

Reply:
version label is the docker build image version..